examples/nifpga/read_write_array_example.py

```diff
@@ -0,0 +1,126 @@
+r""" Read and Write to indicators, controls, Arrays and FIFO.
+
+The gRPC API is built from the C API. NI-FPGA documentation is installed with the driver at:
+    C:\Program Files (x86)\National Instruments\FPGA Interface C API\capi.chm
+
+Getting Started:
+
+To run this example, you need to provide the correct file path and bitfile signature.
```

Comment:
Is there an actual specific example VI/bitfile that this VI is written for? Is that included with a C API (or G API?) example? Or is the intention that the user will create their own (different) FPGA/bitfile and change this script to work with that?

"correct file path and bitfile signature" sounds more like there is a specific "correct" implementation that the user needs to find. If they need to create their own, maybe something more like:

> To run this example, you need to create an FPGA VI and build it into an lvbitx bitfile. Then update this
> example to use the `NI_FPGA_EXAMPLE_BITFILE_PATH ` and `NI_FPGA_EXAMPLE_SIGNATURE` for your
> bitfile. See <FPGA C API documentation?> for more details.

Reply:
Even the C example we provide is similar to this, but we do not include a VI with it. We expect the user to create their own VI and update the code accordingly with the correct values.
Should we include the same comment as we have for the C example?
![image](https://github.com/user-attachments/assets/cc716df6-00d5-487b-936b-2d494c412e0e)

Reply:
Updated comment accordingly.